src/app/Component/RegisterUser/RegisterUser.css

```diff
@@ -124,4 +124,14 @@ label{
     height: 60px;
     justify-content: space-around;
     width: 200px;
+}
+
+.input-change{
```

Comment:
Why another class name  you can import from login.css and you can use the classname .input-change